holidays/countries/gambia.py

```
@@ -0,0 +1,90 @@
+
+if __name__ == "__main__":
+    from datetime import timedelta
+
+    start_year = 2025
+    end_year = start_year + 30
+    all_gambia_holidays = {}
+
+    for year in range(start_year, end_year):
+        gambia_holidays = get_gambia_holidays(year)
+        if gambia_holidays:
+            print(f"\nPublic Holidays in The Gambia for {year}:")
```

Ruff (0.8.2)

`print` found

Remove `print`

(T201)